nvflare/private/fed/server/server_engine.py

```diff
@@ -176,6 +176,8 @@ def heartbeat_to_parent(self):
             except BaseException:
                 # The parent process can not be reached. Terminate the child process.
```

Comment:
should you capture the exception here for logging purpose, so one know that's happening caused the exeception

except BaseException as e: 
    logging.error ( ' ... ')
    raise e;

Reply:
No, this exception is expected. It is used to indicate the corresponding job on the parent side already terminate. It's not an error case.